app/code/Magento/Theme/Block/Html/Footer.php

```diff
@@ -86,6 +93,22 @@ public function getCopyright()
     }
 
     /**
+     * Retrieve Miscellaneous HTML information
+     *
+     * @return string
+     */
+    public function getMiscellaneousHtml()
+    {
+        if (!$this->_misc) {
```

Comment:
As `Magento\Framework\App\Config\ScopeConfigInterface::getValue` returns `mixed` value it's not safe to use this comparison to check is data loaded or not. In such cases (when value may be any type) additional property should be used.